src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1DeployValidPUSHOpcodesFiller.yml

```diff
@@ -0,0 +1,522 @@
+CREATE_EOF1DeployValidPUSHOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 200000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul 
+        {
+          let init_code_len := calldatasize()
+          calldatacopy(0, 0, init_code_len)
+          if iszero(create(0, 0, init_code_len)) { sstore(1, 1) }
+        }
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data: 
+      - ':label push1  :raw 0x7fef00010100030060ff0000000000000000000000000000000000000000000000600052600a6000f3'
```

Comment:
You could have constucted these codes programmatically too, and have one transaction that does a series of 32 creates.

But no need to change as you've already done it this way and it works.

Reply:
I initially tried to do it programatically (see [here](https://gist.github.com/hugo-dc/d69fa1978417f25293a207e8daadd0e5)), but eventually it got too complicated, because there were a lot of things being constructed programmatically: the PUSHn <data>, then convert the resulting code to EOF format, then generate an Initcode for the resulting EOF, so I thought it would be better to go the simpler (although more verbose) way.